esmvaltool/diag_scripts/weighting/weighted_temperature_graph.py

```diff
@@ -77,7 +100,8 @@ def plot_shaded(xrange, upper, lower, color, **kwargs):
     by_label = dict(zip(labels, handles))  # dict removes dupes
     axes.legend(by_label.values(), by_label.keys())
 
-    plt.title('Temperature anomaly relative to 1981-2010')
+    plt.title('Temperature anomaly relative to {}-{}'.format(
+        cfg['settings']['start_year'], cfg['settings']['end_year']))
```

Comment:
I think taking out the start/end year can make this code a bit easier to read. Then we can also define the caption in one go 👍

```suggestion
    start_year = cfg['settings']['start_year']
    end_year = cfg['settings']['end_year']
    caption = f'Temperature anomaly relative to {start_year}-{end_year}'
    plt.title(caption)
```